pymagicc/io.py

```diff
@@ -807,20 +807,27 @@ def read(self):
 
         concs = False
         emms = False
-        if "unit" in metadata:
-            if metadata["unit"] == "ppt":
-                concs = True
-            elif metadata["unit"] == "metric tons":
-                emms = True
-            else:
-                error_msg = "I do not recognise the unit, {}, in file{}".format(
-                    metadata["unit"], self.filepath
+
+        unit_keys = [k for k in metadata.keys() if k.lower() == "unit"]
+        units = [metadata.pop(k) for k in unit_keys]
+        if not units:
+            # have to assume global emissions in tons
+            emms = True
+        elif len(units) != 1:
+            import pdb
```

Comment:
Looks super useful on one hand, but could be very irritating to users not used to Python debugging ...

Reply:
Or is this a left-over?

Reply:
left over, good pick up